src/wesnoth.cpp

```diff
@@ -941,7 +941,7 @@ static void restart_process(const std::vector<std::string>& commandline)
 #define error_exit(res) return res
 #endif
 
-#ifdef __APPLE__
+#ifdef __IPHONEOS__
```

Comment:
Hmm? The dependencies aren't in the repository for the Mac Xcode project. The Headers and lib directories are ignored by git, and can be obtained manually or via a special download from SourceForge (though I'm not sure how up-to-date it's kept).